demo/run-osm-demo.sh

```diff
@@ -24,7 +24,6 @@ BOOKWAREHOUSE_NAMESPACE="${BOOKWAREHOUSE_NAMESPACE:-bookwarehouse}"
 CERT_MANAGER="${CERT_MANAGER:-tresor}"
 CTR_REGISTRY="${CTR_REGISTRY:-localhost:5000}"
 CTR_REGISTRY_CREDS_NAME="${CTR_REGISTRY_CREDS_NAME:-acr-creds}"
-CTR_TAG="${CTR_TAG:-latest}"
```

Comment:
Can we default this to `git rev-parse HEAD` so that users don't have to specify it for local runs.